pkgs/by-name/md/mdbook-tools/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-tools";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "mcmah309";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21